dashboard/test/controllers/report_abuse_controller_test.rb

```diff
@@ -0,0 +1,147 @@
+require 'test_helper'
+require 'cdo/aws/s3'
+
+class ReportAbuseControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  setup do
+    # channels
+    # Use a fake storage ID, because if you use the project_storage factory,
+    # everything breaks.
```

Comment:
thank you for calling this out. this is quite alarming. do you have any idea why? it looks fairly harmless and standard as a factory. if it is this fragile, then we might want to have a warning in comments on the project_storage factory, though that might have to be a bit vague if we don't understand what broke.

Reply:
unfortunately not, I don't understand what's different about this table other than it was apparently part of the pegasus -> dashboard move. this is what was causing unit test failures for projects, api controller, user model, and something else I forget. I can add a comment to the factory saying not to use it in new tests.

Reply:
totally fair, thanks! it does help a bit to understand that adding the factory in this test was causing other tests to break.

Reply:
Ryan, when you get a chance, can you update https://codedotorg.atlassian.net/browse/TEACH-230 (our ticket to track the tech debt with projects testing) with anything you've found? I don't think we'll get to this in the near future but this issue doesn't feel sustainable